### DISCUSSION TITLE
Can you edit?

### DISCUSSION THREAD

**Discussion by ghost** [liriliri/eruda#251] [Q&A]
I was using this, and I think it's really cool, but I want to know if there is a way to edit the code. I like coding with HTML and CSS and I wanted to try to use those here.
If there is any way to edit please tell me.

**Comment by duguowei1000** [comment]
yes i also have the same thinking. Did you get any progress? 

I am thinking of adding chat functionality so this eruda interface is to allow for a typing UI within the browser itself.

**Comment by 775126272** [comment]
> yes i also have the same thinking. Did you get any progress? 
> 
> I am thinking of adding chat functionality so this eruda interface is to allow for a typing UI within the browser itself.
